src/txs/wasm/StoreCodeForm.tsx

```diff
@@ -46,6 +50,7 @@ const StoreCodeForm = ({ chainID }: { chainID: string }) => {
   const tx = {
     estimationTxValues,
     createTx,
+    decimals: readNativeDenom(network[chainID].baseAsset ?? "")?.decimals ?? 6,
```

Comment:
Why aren't these edge case protectors on the other lines of code where the decimals are evaluated?

Reply:
meaning other transaction forms?